Desktop/modules/dynamicmodule.cpp

```diff
@@ -770,26 +641,29 @@ void DynamicModule::setStatus(moduleStatus newStatus)
 	if(_status == moduleStatus::installNeeded && newStatus == moduleStatus::installModPkgNeeded)
 		return;
 
-	_status = newStatus;
+	bool readyForUseInvolved = newStatus == moduleStatus::readyForUse || _status == moduleStatus::readyForUse;
```

Comment:
A pair of `()` makes this nicer to read.

Reply:
Sure